internal/builder/testdata/existing.yml

```diff
@@ -6,8 +6,3 @@ elasticsearch:
       properties:
         example:
           type: short
-      dynamic_templates:
-        - to_nested:
-            path: trigger
-            mapping:
-              type: nested
```

Comment:
If this needs to be removed, please add another one. This file is to be used in a test that ensures that dynamic mappings are added without removing the ones defined in this section.